consensus/hotstuff/verification/common.go

```diff
@@ -43,6 +43,7 @@ func checkVotesValidity(votes []*model.Vote) error {
 
 	// go through all votes to check their validity
 	for _, vote := range votes {
+		// TODO: is checking the view and block id needed? (single votes are supposed to be already checked)
```

Comment:
In fact, the all the checks, which `checkVotesValidity` implements should happen in the component that collects the vote. The purpose of this method is to provide an internal sanity check (which by definition is redundant work). 

Maybe we can move this optimization hint to the logic that calls `checkVotesValidity`, i.e. here: 
https://github.com/onflow/flow-go/blob/541e3acd5aac8e69b62bf7ebc88fdeb682107a1c/consensus/hotstuff/verification/combined_signer.go#L92-L93
and 
https://github.com/onflow/flow-go/blob/541e3acd5aac8e69b62bf7ebc88fdeb682107a1c/consensus/hotstuff/verification/single_signer.go#L94-L95

Reply:
yep, I think this specific sanity check can be skipped.